lms/templates/courseware/progress.html

```diff
@@ -59,15 +59,25 @@ <h1 class="progress-certificates-title">${_("Course Progress for Student '{usern
                     <% post_url = reverse('generate_user_cert', args=[unicode(course.id)]) %>
                     <div class="msg-content">
                         <h2 class="title">${_("Your certificate is available")}</h2>
+                        %if show_cert_web_view:
+                        <p class="copy">${_("You can now view your certificate.")}</p>
+                        %else:
                         <p class="copy">${_(
                             "You can now download your certificate as a PDF. If you keep working and receive a higher grade,you can request an {link_start} updated certificate {link_end}.").format(
```

Comment:
It still applies and because it's in the 'else' block it will only appear if the web view is disabled, yes